tilelang/tools/lower_trace/core.py

```
@@ -197,23 +200,23 @@ class LowerRecord:
-_auto_flush: bool = False
 _script_dir: str | None = None
 _run_dir: str | None = None
 _lock = threading.RLock()
 _run_counter: int = 0
 _atexit_registered: bool = False
+_default_pass_context = None
+_default_pass_context_instruments: list[object] | None = None
+_default_pass_instrument: StackedPassInstrument | None = None
 
 _UNSET: object = object()
 _mode_override: str | None | object = _UNSET
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)